tofu/mag/test_magFieldLines.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+# Also if needed: retab
+'''
+    TEST magFieldLines
+'''
+from __future__ import (unicode_literals, absolute_import,  \
+                        print_function, division)
+import doctest
+import os
+import sys
+
+#print('path 1 =', sys.path)
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+#print('path 2 =', sys.path)
+
+#import mag
+from mag import magFieldLines
+
+doctest.testmod(magFieldLines, verbose=True)
```

Comment:
Not sure how @Didou09 sees this, but... I feel like this test should probably be moved to the `tests` directory?
Whatever the solution chosen, it should take into account that we are going to get rid of `nose`. 
See #232

Reply:
As commented in the review, I suggest:
* Step 1:  minimize changes to system path and restore original system path before the end of the tests, I think we can validate the PR from there
* Step 2: Move the unit tests to a dedicated repo in tofu/tests (create a dedicated issue and PR)
* Step 3: move from nose to something else (see issue #232, but we are not there yet)